### PR TITLE
split couch across multiple machines

### PR DESCRIPTION
**Pull request by snopoke**
@dimagi/scale-team 

goes with https://github.com/dimagi/commcare-hq/pull/15896

### CODE REVIEW

File: ansible/vars/icds/icds_public.yml [hunk 1]
```diff
@@ -223,6 +223,21 @@ site_locations:
     auth_basic: '"Restricted Content"'
     auth_basic_user_file: "/etc/nginx/.htpasswd_adhaar_id_scanner_apk"
 
+couch_dbs:
+  default:
+    host: 10.247.24.18
```
Comment (emord) [inline ansible/vars/icds/icds_public.yml]: Any reason this isn't populated from inventory?

Comment (snopoke) [inline ansible/vars/icds/icds_public.yml]: no, just copied from what was there before

**Comment by emord** [comment]
@dannyroberts feel free to add that to https://github.com/dimagi/commcare-hq/pull/15896 if it'll help you deploy prod/india this week. I'm going to be working off of htese branches when deploying icds

**Comment by dannyroberts** [comment]
@emord oops you're right, I did mean to comment on the other PR. Cool, will do!

**Pull request merged**